tests/package.json

```diff
@@ -4,21 +4,21 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test-with-logs": "mocha  --printlogs -r ts-node/register 'tests/**/*.ts'",
-    "test": "mocha -r ts-node/register 'tests/**/*.ts'"
+    "test-with-logs": "mocha  --printlogs -r ts-node/register 'tests/**/test-*.ts'",
+    "test": "mocha -r ts-node/register 'tests/**/test-*.ts'"
```

Comment:
I don't have a problem with the changes to this file, but they don't seem related to renaming the node binary.

Reply:
Yes, I'll probably revert them